### PR TITLE
Improve performace of mr brain demo

### LINKED ISSUE #361: Demo + bug: unpaired brain MR pre-trained model performance poor
## Subject of the issue

The provided pre-trained network doesn't seem to warp the images fed in as provided in the demo.

If the documentation request is approved, would you be willing to submit a PR?

**Yes** / No

## Your environment

- DeepReg version (commit hash or tag) v. 0.1.6-alpha

- OS (e.g. Ubuntu 20.04, MacOS 10.15, etc.) Mac High Sierra

- Python Version (3.7, 3.8, etc.)   3.7

- TensorFlow 2.3

## Steps to reproduce

Run the demo as indicated for unpaired brain mri, and check the visualisation.

## Expected behaviour

We would expect the pre-trained registration network to warp the moving image to resemble the fixed image. 

## Actual behaviour

As in Issue #347 the performance of the pre-trained model is strange - the pred-fixed-image (the warped moving image) and labels are not warped (appear the same), if we look at the visualisation.

##  Suggested fix

Train a model for longer...?

### PR DESCRIPTION
**Pull request by acasamitjana**
# Description

I retrained the model with different hyperparameters and for a longer period of time. I updated the demo_train and demo_predict accordingly, as well as upload the last model to the zoo.

Fixes #361 

## Type of change

What types of changes does your code introduce to DeepReg? _Put an `x` in the boxes that
apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Documentation Update (fix or improvement on the documentation)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Other (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR.
If you're unsure about any of them, don't hesitate to ask. We're here to help! This is
simply a reminder of what we are going to look for before merging your code._

- [x] I have
      [installed pre-commit](https://deepreg.readthedocs.io/en/latest/contributing/setup.html)
      and formatted all changed files.
- [x] My commits' message styles matches
      [our requested structure](https://deepreg.readthedocs.io/en/latest/contributing/commit.html),
      e.g. `Issue #<issue number>: detailed message`.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by mathpluscode** [comment]
@acasamitjana ^^ you are trying to merge it into main, maybe you want to do it with dev?

**Comment by acasamitjana** [comment]
> @acasamitjana ^^ you are trying to merge it into main, maybe you want to do it with dev?

Ups, correct. Sorry.

**Comment by mathpluscode** [comment]
@acasamitjana oh you could have modified it directly, no need to close the PR. But anyway, it's not important.
